tests/common/get_cts_string.h

```diff
@@ -0,0 +1,161 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides tools for string representation for some types
+//
+*******************************************************************************/
+
+#ifndef __SYCLCTS_TESTS_COMMON_GET_CTS_STRING_H
+#define __SYCLCTS_TESTS_COMMON_GET_CTS_STRING_H
+
+#include "common.h"
+
+namespace sycl_cts {
+namespace get_cts_string {
+
+enum class verbosity { brief = 0, detail };
+
+/** @brief Stringify bool value
+ *  @tparam bool flag Value to convert
+ *  @retval String with interpretation of bool
+ */
+inline std::string for_bool(bool flag) { return flag ? "true" : "false"; }
```

Comment:
It seems to me like most (if not all) of these functions could be `constexpr` and return `std::string_view`